src/usdm-client.ts

```diff
@@ -176,13 +176,7 @@ export class USDMClient extends BaseRestClient {
     return this.get('fapi/v1/premiumIndexKlines', params);
   }
 
-  getMarkPrice(params: BasicSymbolParam): Promise<MarkPrice>;
-
-  getMarkPrice(): Promise<MarkPrice[]>;
```

Comment:
Same here with the overrides, please keep them so they resolve the array vs non array return type.

Reply:
See my other comments about the overrides, similar approach should do the trick